runtime/src/configs/mod.rs

```diff
@@ -444,6 +444,7 @@ impl pallet_randomness::Config for Runtime {
 parameter_types! {
     pub const SpMinDeposit: Balance = 100 * UNIT;
     pub const BucketDeposit: Balance = 100 * UNIT;
+    pub const BspSignUpLockPeriod: BlockNumber = 90 * DAYS; // ~3 months
```

Comment:
Maybe 3 months is a bit too much? Not opposed to it though

Reply:
The auditor had suggested a year...